src/pages/articles/bootstrap/bootstrap-get-started/index.md

```diff
@@ -2,14 +2,47 @@
 title: Bootstrap Get Started
 ---
 ## Bootstrap Get Started
+When using Bootstrap for the first time it can appear to be a confusing process, but using Bootstrap is actually quite simple. Using Bootstrap makes it easy to design a fully responsive website and is a framework worth learning.
 
-This is a stub. [Help our community expand it](https://github.com/freecodecamp/guides/tree/master/src/pages/articles/bootstrap/bootstrap-get-started/index.md).
+#### What is a responsive website:
+A responsive website is a website that resizes and rearranges that items on the page depending on the size of your browser. With a responsive website if you resize your browser you can see the changes occur in real time. Bootstrap makes your website responsive for you.
 
-[This quick style guide will help ensure your pull request gets accepted](https://github.com/freecodecamp/guides/blob/master/README.md).
+#### How do I add Bootstrap to my page:
+Adding bootstrap to your page is quite simple, you simply add the following to the `<head>` tags in your code.
+```
+<!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+<!-- Latest compiled and minified JavaScript -->
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+```
 
-#### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
+You will also need to add the following between the `body` tags in your code. With bootstrap you'll be using `<div>` tags when using many of Bootstrap's features, each tag will have it's own unique set of classes applied that allows the tag to perform it's task. Other sections of this Bootstrap guide will show more examples of how Bootstrap uses `<div>` tags. (`<div>` tags are not exclusive to Bootstrap however Bootstrap makes use of them.). Below is the code that would would add to the `body` tags in your code to finish getting started. Keep in mind that while this creates the container, the page will still remain blank until you add content to the container.
+```
+<div class="container-fluid">
+
+</div>
+```
 
+#### Full page code example:
```

Comment:
I don't think this will make it clearer. I think we can safely assume that if someone is looking for information on how to use Bootstrap, they have the basics of a web page already